### PR TITLE
util/codec: unify decimal codec with tikv.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@coocood  @shenli  @BusyJay PTAL

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: util/codec/decimal.go [hunk 1]
```diff
@@ -129,6 +128,9 @@ func DecodeDecimal(b []byte) ([]byte, mysql.Decimal, error) {
 		return r, d, errors.Trace(err)
 	}
 
+	// Set decimal point for value.
+	value = append([]byte("0."), value...)
```
Comment (BusyJay) [inline util/codec/decimal.go]: move to L138 seems better?

Comment (qiuyesuifeng) [inline util/codec/decimal.go]: Seems not, we must append sign symbol ahead of value bytes.

**Comment by qiuyesuifeng** [comment]
@BusyJay PTAL

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**